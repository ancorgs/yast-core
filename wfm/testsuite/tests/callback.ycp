{
  // callback.ycp

  integer a = 42;

  integer wfm_result = UI::``{ integer a = 13; return WFM(``{ return a;}); };
  string ui_scr_result = UI::``{ return SCR(`Read(.probe.architecture)); };
  string wfm_scr_result = SCR::`Read(.probe.architecture);

  return [wfm_result==a, ui_scr_result == wfm_scr_result];
}
