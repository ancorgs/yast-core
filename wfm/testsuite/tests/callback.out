({
  integer `a = 13;
  return WFM::``{
  return a;
}
;
}
)
({
  return SCR::`Read (.probe.architecture);
}
)
([false, true])
